content/zh/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -297,6 +289,91 @@ kubectl 为 Bash 和 Zsh 提供自动补全功能，可以减轻许多输入的
 {{< tab name="Zsh" include="included/optional-kubectl-configs-zsh.md" />}}
 {{< /tabs >}}
 
+<!--
+### Install `kubectl convert` plugin
+-->
+### 安装 `kubectl convert` 插件
+
+{{< include "included/kubectl-convert-overview.md" >}}
```

Comment:
This line is blocking the CI. We don't have the file yet.

Reply:
should I remove this line from doc?

Reply:
No. That file should be copied from the English upstream and translated.